webapp/components/profile_popover.jsx

```diff
@@ -143,32 +141,20 @@ export default class ProfilePopover extends React.Component {
                 );
             }
 
-            const webrtcTooltip = (
-                <Tooltip id='webrtcTooltip'>{webrtcMessage}</Tooltip>
-            );
-
             webrtc = (
                 <div
-                    className='webrtc__user-profile'
+                    data-toggle='tooltip'
                     key='makeCall'
+                    className='popover__row'
                 >
                     <a
                         href='#'
+                        className='text-nowrap text-lowercase user-popover__email'
```

Comment:
seems "text-lowercase" has no effect here because we didn't use lowercase in this link.